roles/cis/tasks/3/verify/3.1.1.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: "3.1.1 Verify if IP forwarding is disabled (Scored)"
+  block:
+    - name: "3.1.1 verify if net.ipv4.ip_forward is set to 0"
+      command: grep -Fxq "net.ipv4.ip_forward=0" /etc/sysctl.conf
+      register: output_1
+    - fail:
+        msg: "3.1.1 failed to verify if IP forwarding is disabled, net.ipv4.ip_forward is not set to 0"
+      when: output_1.rc != 0
+    - name: "3.1.1 verify if net.ipv6.conf.all.forwarding is set to 0"
+      command: grep -Fxq "net.ipv6.conf.all.forwarding=0" /etc/sysctl.conf
```

Comment:
This check wont be sufficient for verifying whether ip_forward is disabled. Even though this value is in /etc/sysctl.conf there might be cases where this value is not properly applied. Better way is to check the runtime proc values or use sysctl command to query for this and check the return value.